### ISSUE TITLE
Custom language

### ISSUE THREAD

**Issue by alvarodevlabs** [shahraza/doo.custom#1]
Fist of all thanks for the share,

Just one question, it is posibble to select torrents language? because i need it all in Spanish.

Thanks!!

**Comment by shahraza** [comment]
Can you introduce the website that the Spanish Torrent offers?

**Comment by alvarodevlabs** [comment]
Yeah sure, something like https://grantorrent.net/ is a great website to scrape, so much simple.
Or another one https://mejortorrent1.net/peliculas-hd/

**Comment by xsouf** [comment]
D

**Comment by AnesSama** [comment]
it is posibble to select torrents language (french)?
